python/chronos/src/bigdl/chronos/utils.py

```diff
@@ -30,3 +32,67 @@ def wrapped(*args, **kwargs):
             return function(*args, **kwargs)
         return wrapped
     return deprecated_decorator
+
+
+# code adaptted from https://github.com/intel/neural-compressor/
+#                    blob/master/neural_compressor/utils/utility.py#L62
+class LazyImport:
+    """
+    Example:
+        >>> from bigdl.chronos.utils import LazyImport
+        >>> forecaster = LazyImport('bigdl.chronos.forecaster.lstm_forecaster.LSTMForecaster')
```

Comment:
just to confirm, so the `pkg` parameter won't be used right?

Reply:
Yep.